code/lib/cli/src/initiate.ts

```diff
@@ -405,7 +405,8 @@ export async function doInitiate(options: CommandOptions): Promise<
   );
 
   return {
-    shouldRunDev: process.env.CI !== 'true' && process.env.IN_STORYBOOK_SANDBOX !== 'true',
+    shouldRunDev:
+      options.dev ?? (process.env.CI !== 'true' && process.env.IN_STORYBOOK_SANBOX !== 'true'),
```

Comment:
since there is direct programatic usage of `doInitiate` in the `sandbox` task, and because of the Commander behaviors around boolean options mentioned above, I ended up needing to define the default behavior with the environment variables in two places (in the Commander program definition and in the code itself). If anyone has a better idea for this, let me know!